ui/common/src/socket.ts

```diff
@@ -41,47 +41,50 @@ interface Settings {
   params?: Partial<Params>;
   options?: Partial<Options>;
 }
-
-const origSend = WebSocket.prototype.send;
+// TODO - find out why are there three different types of settings
 
 const isOnline = () => !('onLine' in navigator) || navigator.onLine;
 
-// versioned events, acks, retries, resync
-export default class StrongSocket {
```

Comment:
- tie public method typings to the implementation (via extending `SocketI` which is defined in `index.d.ts`)
- make non-public members private (more as an annotation than access restriction, as they are not available in `SocketI` anyway)